ComponentKit/Debug/CKComponentHierarchyDebugHelper.mm

```diff
@@ -52,6 +52,9 @@ + (NSString *)componentHierarchyDescriptionForView:(UIView *)view searchUpwards:
 
 + (NSString *)componentHierarchyDescriptionForView:(UIView *)view searchUpwards:(BOOL)upwards showViews:(BOOL)showViews
 {
+  if ([view isKindOfClass:[CKComponent class]]) {
+    view = ((CKComponent *)view).viewContext.view;
+  }
```

Comment:
This is hacky. The functionality you care about can likely be implemented fully in chisel where you can do the same checks to get the view. It might be a bit less convenient to evaluate commands in python, but definitely doable.